raiden/constants.py

```diff
@@ -87,6 +87,12 @@ class Capabilities(Enum):
     TODEVICE = "toDevice"  # supports sending and receiving toDevice messages on Matrix
 
 
+class DeviceIDs(Enum):
```

Comment:
I would expec this to be singular
```suggestion
class DeviceID(Enum):
```

Reply:
I just styed consistent with other types such as `networks` and `Capabilities`

Reply:
We also have `ServerListType`, but let's go with the majority of existing enums and use the plural, then.